applications/matter_bridge/src/bridged_device_types/generic_switch.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include "matter_bridged_device.h"
+
+class GenericSwitchDevice : public MatterBridgedDevice {
+public:
+	GenericSwitchDevice(const char *nodeLabel);
+
+	MatterBridgedDevice::DeviceType GetDeviceType() const override
+	{
+		return MatterBridgedDevice::DeviceType::GenericSwitch;
+	}
+	CHIP_ERROR HandleRead(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer,
+			      uint16_t maxReadLength) override;
+	CHIP_ERROR HandleAttributeChange(chip::ClusterId clusterId, chip::AttributeId attributeId, void *data,
+					 size_t dataSize) override;
+	CHIP_ERROR HandleWrite(chip::ClusterId clusterId, chip::AttributeId attributeId, uint8_t *buffer) override
+	{
+		return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+	};
+
+private:
+	CHIP_ERROR HandleReadSwitch(chip::AttributeId attributeId, uint8_t *buffer, uint16_t maxReadLength);
+
+	static constexpr uint16_t GetSwitchClusterRevision() { return 1; }
+	static constexpr uint32_t GetSwitchClusterFeatureMap() { return 1; }
```

Comment:
Shouldn't it be 2 (MS), instead of 1 (LS)? Also maybe it would be worth to add comment what this feature map value means. I know we have not done it before, but now I see it would be good to do that.

Reply:
Ahh of course, it's bit 1 in the case of our implementation.